icu4c/source/python/icutools/databuilder/__main__.py

```diff
@@ -237,6 +238,25 @@ def add_copy_input_requests(requests, config, common_vars):
     return result
 
 
+class IO(object):
+    """I/O operations required when computing the build actions"""
+
+    def __init__(self, src_dir):
+        self.src_dir = src_dir
+
+    def glob(self, pattern):
+        result_paths = pyglob.glob(os.path.join(self.src_dir, pattern))
+        # For the purposes of icutools.databuilder, force Unix-style directory separators.
+        return [v.replace("\\", "/")[len(self.src_dir)+1:] for v in sorted(result_paths)]
```

Comment:
Is there no "proper" way to handle Windows style directory paths in Python?
I see you appear to be trying to limit the '\' replacement to path segments up until the src_dir length, or something.
Is this still safe in the face of a Unix file with a '\' in?
Perhaps a comment explaining the approach would be good (or is it simpler to convert src_dir first since no patterns should have '\\' in)?